src/main/java/com/example/travelservice/dtoconverters/TripConverter.java

```diff
@@ -36,14 +32,15 @@ public Trip convertTripDtoToTrip (TripDto source) {
                 Enum.valueOf(Meal.class, source.getMeal()));
     }
 
-    public WeatherRequest convertTripToWeatherRequest (Trip source) {
+    public WeatherRequest convertTripToWeatherRequest (Trip source, Long tripId) {
```

Comment:
why you need tripId here if you have trip itself?

Reply:
resolved